ResidentInformationApi/V1/Domain/Mosaic/MosaicResidentInformationResult.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+
+namespace ResidentInformationApi.V1.Domain
+{
+    public class MosaicResidentInformationResult
+    {
+        /// <example>
+        /// academy
+        /// </example>
+        public string System { get; set; }
+        /// <example>
+        /// 123_456
+        /// </example>
+        public string SystemId { get; set; }
+        /// <example>
+        /// https://academy-api.hackney.gov.uk
```

Comment:
I thought this would be the link to a specific record, not just the academy API in general, but might be wrong? If so it would be good to see that reflected int he example

Reply:
My understanding is that we return the result object which contains the name of the system and then a list of all the contact information returned from the system. Though we're somewhat working in the dark as a lot of this was designed before the SwaggerDoc was finalised